google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ToProtoConverter.java

```diff
@@ -17,11 +17,18 @@
 
 import com.google.protobuf.Descriptors;
 import com.google.protobuf.DynamicMessage;
+import java.util.List;
 
 public interface ToProtoConverter<T> {
   DynamicMessage convertToProtoMessage(
       Descriptors.Descriptor protoSchema,
       TableSchema tableSchema,
-      T inputObject,
```

Comment:
Thinking more about this, since adding a new method to interface is already a breaking change, let's just remove the old method, sorry.

Reply:
Adding a new method is not a breaking change for the user who is using ToProtoConverter but not implementing it, which I think is the majority of user's using pattern?

Reply:
If you added a new method, there is no way they will have implementations for it? So their code will fail to build.

Reply:
Done